x-pack/plugins/lens/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "author": "Elastic",
+  "name": "visualization_lens",
```

Comment:
The plugin id is `visualization_editor` the product name is `Lens Visualizations` and the name of the package `visualization_lens`
What is the correct name for that?

Reply:
It's lens at the top level, but we want it to be easily changed